src/container/Home.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Home.css';
+
+const Home = () => (
+  <div className="Home">
+    <h1>Welcome to our page!</h1>
+    <p>
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+    </p>
+    <p>Esse neque explicabo quod eligendi necessitatibus beatae cumque tenetur.</p>
+    <p>Officiis, excepturi illum eos vel delectus quod? </p>
```

Comment:
- [ ] [OPTIONAL] Instead of using the placeholder text `lorem ipsum...` for your welcome page, I'll suggest you to customize it with a text related to your web application, something about mathematics, or a calculator.